latch_cli/main.py

```diff
@@ -206,8 +207,8 @@ def init(
 
 
 @main.command("cp")
-@click.argument("src", nargs=1)
-@click.argument("dest", nargs=1)
+@click.argument("src")
+@click.argument("dest")
 @click.option(
     "--max-concurrent-files",
     "--mcf",
```

Comment:
needs a name that conveys what it actually does
like since this is for progress bars maybe it should be like
`--progress-simultaneous-tasks` or whatever

in general just seems weird. why would you ever change it? it should just show exactly as many bars as there are actual concurrent tasks instead

Reply:
nixed